_stories/utilities/Float/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { selectV2 as select, object } from '@storybook/addon-knobs';
+import { select } from '@storybook/addon-knobs';
```

Comment:
this component gives me an object is not defined error when i go to it in storybook
oh, it's missing object import here